pkg/operator/k8sutil/network.go

```diff
@@ -97,6 +107,9 @@ func ApplyMultus(net cephv1.NetworkSpec, objectMeta *metav1.ObjectMeta) error {
 		return fmt.Errorf("ApplyMultus: Can't mix short and JSON form")
 	}
 
+	// Sort network strings so that pods/deployments won't need updated in a loop if nothing changes
+	sort.Strings(v)
```

Comment:
This seems to be the fix proposed by the commit, other changes seem more improvements, right? If so, please decouple into individual commits.

Reply:
What about making the unit test changes part of the sort commit? Then we can merge the sorting change with tests, and perhaps save the other changes for another PR.